README.md

```diff
@@ -499,3 +504,12 @@ a module that can do that.
 Open an [issue](https://github.com/solarkennedy/puppet-consul/issues) or
 [fork](https://github.com/solarkennedy/puppet-consul/fork) and open a
 [Pull Request](https://github.com/solarkennedy/puppet-consul/pulls)
+
+
+## Transfer Notice
+
+This plugin was originally authored by [KyleAnderson](http://github.com/KyleAnderson).
```

Comment:
Plugin?

Reply:
I'm totally fine without this notice here. My username is `solarkennedy` anyway, which makes it not even right! :)

Reply:
was a bit confusing that the forge username is different to the GitHub handle :D I will update it